styles.css

```diff
@@ -0,0 +1,17 @@
+body {
+  margin: 0;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  height: 15%;
+  background-color: #fefbe9;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+h1 {
+  margin: 0;
+}
```

Comment:
By applying `margin: 0;` within the `*` selector, you eliminate the need to specify this rule individually for each element. This simplifies your stylesheet and ensures a consistent starting point for the layout of all elements on your page

```suggestion

```